source/user-manual/certificates.rst

```diff
@@ -67,11 +64,11 @@ After running the script, the directory ~/certs will be created and will have th
         certs/
```

Comment:
This folder is `wazuh-certificates` now.

Reply:
Done: https://github.com/wazuh/wazuh-documentation/commit/3c5a90c58931d83484826b0f85c8e0c657ce2d18